src/house.h

```diff
@@ -22,6 +22,7 @@
 
 #include <regex>
 #include <set>
+#include <unordered_set>
```

Comment:
what's the reason behind this?

Reply:
unordered_set was previously implicitly imported through tile.h, since it is no longer imported in tile.h, it has to be imported in house.h because house.h uses unordered_sets.